.codeclimate.yml

```diff
@@ -0,0 +1,162 @@
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
+
+version: "2"
+
+plugins:
+
+# ---------------
+# Cross-language plugins. Should always be on.
+
+  duplication: # Looks for similar and identical code blocks
+    enabled: true
+    config:
+      languages:
+        go:
+        java:
+        javascript:
+        php:
```

Comment:
I don't think we use php anywhere (nor will we)

Reply:
Yep, we do (though it might be an archivable repo at this point?). conjurinc/conjur-wordpress-theme has it. I was surprised to find that across our repos we have 25+ different programming languages (for the languages that GH shows in the colored bar above the code listing on the project overview page). It's 25+ because 12 repos have "Other" listed and unlike the specific languages, GH doesn't link that to a file list, so I'm not sure what "Other" actually means.